music21/romanText/tsvConverter.py

```diff
@@ -929,33 +1053,10 @@ def characterSwaps(preString, minor=True, direction='m21-DCML'):
     for thisKey in characterDict:  # Both major and minor
         preString = preString.replace(thisKey, characterDict[thisKey])
 
-    if not minor:
-        return preString
-    else:
-        if direction == 'm21-DCML':
-            search = 'b'
-            insert = '#'
-        elif direction == 'DCML-m21':
-            search = '#'
-            insert = 'b'
-        m = re.search('vii?', preString)
-        if m is not None:
-            # Previously, this function here matched VII and vii but not vi; for V2
-            # (at least), we need to match vii and vi but *not* VII; this version
-            # also passes V1 tests.
-            position = m.start()
-            prevChar = preString[position - 1]  # the previous character,  # / b.
-            if prevChar == search:
-                postString = preString[:position - 1] + preString[position:]
-            else:
-                postString = preString[:position] + insert + preString[position:]
-        else:
-            postString = preString
+    return preString
 
-    return postString
 
-
-def getLocalKey(local_key, global_key, convertDCMLToM21=False):
+def getLocalKey(local_key: str, global_key: str, convertDCMLToM21: bool = False):
```

Comment:
type return value, especially important since this does not return a key.Key object as implied.